src/styles/index.scss

```diff
@@ -1,5 +1,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&family=Open+Sans:wght@400;500;600;700;800&family=Poppins:wght@400;500;600;700;800&family=Roboto:wght@300;400;500;700;900&display=swap');
 
+/* stylelint disable  Unknown property */
```

Comment:
- [ ] OPTIONAL
You have **disabled Stylelint check** on parts of your code. Please enable all Stylelint checks on your code and try modifying that portion of your code to adhere to Stylelint standards. This will ensure a robust  and clean CSS code.

Reply:
Thank you @rloterh